backend/src/user/controllers/getUserController.ts

```diff
@@ -0,0 +1,5 @@
+import { Request, Response } from "express";
+
+export async function getUserController(req: Request, res: Response, next: Function) {
```

Comment:
get user controller means it returns a controller function but it's already the controller, itself.